x-pack/solutions/observability/plugins/synthetics/public/apps/synthetics/components/alerts/rule_viz.tsx

```diff
@@ -99,18 +117,17 @@ export const RuleViz = ({ dispatchedAction }: { dispatchedAction: PayloadAction<
         {/* to push detail button to end*/}
         <EuiFlexItem />
         <EuiFlexItem grow={false}>
-          <EuiButtonEmpty
-            data-test-subj="syntheticsRuleVizInspectButton"
-            onClick={inspect}
-            iconType="inspect"
-            size="xs"
-          >
-            {i18n.translate('xpack.synthetics.rules.details', {
-              defaultMessage: 'Details',
-            })}
-          </EuiButtonEmpty>
+          {isInspectorEnabled ? (
+            detailsButton
+          ) : (
+            <EuiToolTip content={inspectorDisabledTooltip}>{detailsButton}</EuiToolTip>
+          )}
         </EuiFlexItem>
       </EuiFlexGroup>
     </EuiCallOut>
   );
 };
+
+const inspectorDisabledTooltip = i18n.translate('xpack.synthetics.rules.inspectorDisabled', {
+  defaultMessage: 'Enable "Inspect ES queries" in Advanced Settings to see the Details',
```

Comment:
@bmorelli25 Is this wording clear? (More info in PR description)

Reply:
Small nit: I would remove the word "the" here.

```suggestion
  defaultMessage: 'Enable "Inspect ES queries" in Advanced Settings to see Details',
```

Reply:
Changed [here](https://github.com/elastic/kibana/pull/216824/commits/a7634fe14bba777afc437d5a015f7a14ad96a7a0).

Thanks!